packages/api/src/libs/action-client.ts

```diff
@@ -0,0 +1,40 @@
+import fetch from "isomorphic-fetch";
+import type { Json } from "..";
+import assert from "assert";
+import join from "url-join";
+
+// This is a basic client to allow you to call into the server in the form of
+// client(action,arg1,arg2,...etc) => Promise<Json>.
+// see tests for usage.
+export default function Client(host: string) {
```

Comment:
@Gamaranto i added you to review just to see this client, which u can use as a reference implementation to talk to server.  currently no way to export this to you yet.